### PR TITLE
Adding model solutions to exercises

### PR DESCRIPTION
**Pull request by Mirasalm**
Work in progress branch for more model solutions. Currently has some for serial and monitoring tutorial. I'll continue iterating on current ones and adding more, but any feedback is appreciated.

### CODE REVIEW

**Comment by Mirasalm** [comment]
I think these are good enough for review/merge, but it's probably good if someone can at least take a glance and see if there are any mistakes.

File: triton/tut/dependency.rst [hunk 1]
```diff
@@ -47,8 +47,23 @@ Exercises
    Create a chain of jobs A -> B -> C each depending on the successful
    completion of the previous job.  In each job run e.g. ``sleep 60``
    to give you time to investigate the status of the queue.
+   
+   .. solution::
+   
+      You should all of your jobs in queue. Jobs with dependency on a
```
Comment (rkdarst) [inline triton/tut/dependency.rst:53]:
```suggestion
      You should chain all of your jobs in queue. Jobs with dependency on a 
```

**Review by rkdarst — APPROVED** [review]
Looks good to me!

**Comment by Mirasalm** [comment]
Some storage questions compare find and lfs find. Based on my testing lfs find seems to actually be slower. Is it just demo effect in action or is that part of tutorials out of date and lfs find is unnecessary nowadays?

**Pull request merged**